tests/unit/test_util.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
I would have loved to see unit tests testing the `AfflitiateMemory` and `cache_validity` of the `CachingAffiliateFetcher` class.